semcore/spin-container/CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [7.8.0] - 2023-09-19
```

Comment:
JFYI: it's not needed to update changelogs if you are not changing component source code

Website documentation is made to work in kind of alongside way. It's republished after each release or by manual triggering [the website publish workflow](https://github.com/semrush/intergalactic/actions/workflows/publish-website.yml)

Reply:
removed in all MRs